otaserver/coap.py

```diff
@@ -91,14 +91,15 @@ def render_get(self, request):
 class CoapController():
     """CoAP controller with CoAP server inside."""
 
-    def __init__(self, fw_path, port=COAP_PORT):
+    def __init__(self, fw_path, port=COAP_PORT, dtls_enabled=False):
```

Comment:
I would rename `dtls_enabled` to `use_dtls` or `use_coaps`. What do you think ?

Reply:
For this one, I'm not inclined to do it. This is mostly because the way aiocoap is handling the CoAP(s) packets, and the use of names in that module.

Reply:
With your last changes, I tend to agree with you. Let's keep this one.